code/modules/atmospherics/machinery/components/fusion/hfr_fuel_datums.dm

```diff
@@ -0,0 +1,127 @@
+///Global list of recipes for atmospheric machines to use
+GLOBAL_LIST_INIT(hfr_fuels_list, hfr_fuels_create_list())
+
+/*
+ * Global proc to build the gas recipe global list
+ */
+/proc/hfr_fuels_create_list()
+	. = list()
+	for(var/fuel_mix_path in subtypesof(/datum/hfr_fuel))
+		var/datum/hfr_fuel/fuel_mix = new fuel_mix_path()
+
+		.[fuel_mix.id] = fuel_mix
+
+/datum/hfr_fuel
+	///Id for the mix
+	var/id = ""
+	///The gases that are going to be used as fuel ("GasX + GasY fuel")
+	var/name = ""
+	///Multiplier for the minimum heat output of the HFR (min 0.01)
+	var/negative_temperature_multiplier = 1
+	///Multiplier for the maximum heat output of the HFR (min 0.01)
+	var/positive_temperature_multiplier = 1
+	///Multiplier for the energy released (min 0.01)
+	var/energy_concentration_multiplier = 1
+	///Multiplier for the fuel consumption (min 0.01)
+	var/fuel_consumption_multiplier = 1
+	///Multiplier for the gas production (min 0.01)
+	var/gas_production_multiplier = 1
+
+	var/temperature_change_multiplier = 1
+	///Only 2 gases
+	var/requirements = list()
+	///Gases that gets produced directly in the internal gasmix
+	var/primary_products = list()
+	///Gases that gets produced in the moderator gasmix or directly ejected (must be 6 gases)
+	var/secondary_products = list()
```

Comment:
Does the order matter here? if so, in what way?

Reply:
i'll specify that the order matter because of different power level different gases will be produced, the first gas will be made in power level 1 and 2, the second in power level 2 and 3, and so on.

So the order of these gases will let you select when the gases are going to get produced on the power level scale

Reply:
👍